### DISCUSSION TITLE
Allow fields to span in the third column of detail view

### DISCUSSION THREAD

**Discussion by AlexVanK** [directus/directus#10015] [Feature Requests]
I was able to achieve this on Directus 8 with some custom CSS and JS
![image](https://user-images.githubusercontent.com/56647386/143107871-7d8e3d10-dcd7-40b3-bc08-702732aec9a1.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
